packit/cli/dist_git_init.py

```diff
@@ -240,6 +250,18 @@ def init(
         upstream_git_url = result.stdout.strip()
         click.echo(f"Found the following URL: {upstream_git_url}")
 
+    dist_git_branches_mapping_dict = {}
+    if dist_git_branches_mapping:
+        try:
+            dist_git_branches_mapping_dict = json.loads(dist_git_branches_mapping)
```

Comment:
I would add `isinstance(dist_git_branches_mapping_dict, dict)` - for example `[]` is also valid json.